README.md

```diff
@@ -1,50 +1,67 @@
 # SignalFx .NET Lambda Wrapper
 
-## Overview
+The SignalFx .NET Lambda Wrapper wraps around an AWS Lambda .NET or ASP.NET Core
+function handler, which enables you to send metrics and traces to Splunk APM.
 
-You can use this document to add a SignalFx wrapper to your AWS Lambda for .NET.
+The Lambda wrapper manually instruments the Lambda function itself, and not
+any libraries or frameworks. To get traces from libraries and frameworks in
+the Lambda function, manually instrument them.
 
-The SignalFx C# Lambda Wrapper wraps around an AWS Lambda .NET function handler, which allows metrics and traces to be sent to SignalFx.
+See these links for information about and examples of manual instrumentation:
 
-At a high-level, to add a SignalFx .NET Lambda wrapper, you can:
-   * Package the code yourself; or
-   * Use a Lambda layer containing the wrapper, and then attach the layer to a Lambda function.
+- https://github.com/signalfx/tracing-examples/tree/master/dotnet-manual-instrumentation
+- https://github.com/signalfx/tracing-examples/tree/master/signalfx-tracing/signalfx-dotnet-tracing
```

Comment:
The second link is an example mixing manual and auto-instrumentation. Keep only the first one.